cabal.project

```diff
@@ -0,0 +1,5 @@
+packages: bytestring.cabal
+allow-newer: *:bytestring
```

Comment:
Surely there can't be any reverse dependencies of bytestring involved here?

Reply:
When testing with GHC-head, `text`, et. al. needed for the tests want an older `bytestring` than the one we're testing, but we want to force them to use the new one.

Reply:
But this `cabal.project` doesn't include the tests?!